support-workers/src/typescript/lambdas/createPaymentMethodLambda.ts

```diff
@@ -0,0 +1,194 @@
+import { combinedAddressLine } from '../model/address';
+import type {
+	DirectDebitPaymentFields,
+	PaymentFields,
+	PayPalPaymentFields,
+	StripePaymentFields,
+} from '../model/paymentFields';
+import type {
+	DirectDebitPaymentMethod,
+	PaymentMethod,
+	PayPalPaymentMethod,
+	StripePaymentMethod,
+} from '../model/paymentMethod';
+import { stageFromEnvironment } from '../model/stage';
+import type {
+	CreatePaymentMethodState,
+	CreateSalesforceContactState,
+	User,
+	WrappedState,
+} from '../model/stateSchemas';
+import {
+	createPaymentMethodStateSchema,
+	wrapperSchemaForState,
+} from '../model/stateSchemas';
+import { ServiceHandler } from '../services/config';
+import { getPayPalConfig, PayPalService } from '../services/payPal';
+import { getStripeConfig, StripeService } from '../services/stripe';
+import { getIfDefined } from '../util/nullAndUndefined';
+
+const stage = stageFromEnvironment();
+const stripeServiceHandler = new ServiceHandler(stage, async (stage) => {
+	const config = await getStripeConfig(stage);
+	return new StripeService(config);
+});
+const paypalServiceHandler = new ServiceHandler(stage, async (stage) => {
+	const config = await getPayPalConfig(stage);
+	return new PayPalService(config);
+});
+
+export const handler = async (
+	state: WrappedState<CreatePaymentMethodState>,
+) => {
+	console.info(`Input is ${JSON.stringify(state)}`);
+	const createPaymentMethodState = wrapperSchemaForState(
+		createPaymentMethodStateSchema,
+	).parse(state).state;
+	return createSalesforceContactState(
+		createPaymentMethodState,
+		await createPaymentMethod(
+			createPaymentMethodState.paymentFields,
+			createPaymentMethodState.user,
+		),
+	);
+};
+
+export function createPaymentMethod(
+	paymentFields: PaymentFields,
+	user: User,
+): Promise<PaymentMethod> {
+	switch (paymentFields.paymentType) {
+		case 'Stripe':
+			return createStripePaymentMethod(user.isTestUser, paymentFields);
+		case 'PayPal':
+			return createPayPalPaymentMethod(user.isTestUser, paymentFields);
+		case 'DirectDebit':
+			return createDirectDebitPaymentMethod(paymentFields, user);
```

Comment:
keep the params a similar way around?
```suggestion
			return createDirectDebitPaymentMethod(user, paymentFields);
```